torch_geometric/nn/aggr/multi.py

```diff
@@ -19,7 +20,18 @@ def __init__(self, aggrs: List[Union[Aggregation, str]]):
             raise ValueError(f"'aggrs' of '{self.__class__.__name__}' should "
                              f"not be empty")
 
-        self.aggrs = [aggregation_resolver(aggr) for aggr in aggrs]
+        if aggrs_kwargs is None:
+            aggrs_kwargs = [{}] * len(aggrs)
+        elif len(aggrs) != len(aggrs_kwargs):
+            raise ValueError(f"'aggrs_kwargs' with invalid length passed to "
```

Comment:
This error seems like something pretty generic. Do you think there is any value in having a util to generate such an error?

Reply:
Fixed.